samples/java/src/main/java/com/contoso/kafka/TriggerFunction.java

```diff
@@ -38,10 +38,11 @@ public void runMany(
                 topic = "message", 
                 brokerList="%BrokerList%",
                 consumerGroup="$Default", 
-                username = "%UserName%", 
-                password = "%Password%",
+                username = "%ConfluentCloudUsername%", 
+                password = "%ConfluentCloudPassword%",
                 authenticationMode = BrokerAuthenticationMode.PLAIN,
                 protocol = BrokerProtocol.SASLSSL,
+                sslCaLocation = "confluent_cloud_cacert.pem", // Remove this attribute for Linux/Mac users.
```

Comment:
Can we please check removing it with not throw any warnings?

Reply:
For the windows, It could be a problem. Windows has the entry on Registry. However, .NetCore can't see the registry. That is why we need to add pem only on windows.